### DISCUSSION TITLE
Healthy Federated Data Marketplace

### DISCUSSION THREAD

**Discussion by kolaisaac** [OpenMined/courses#249] [Feedback] [answered]
Special thanks to @iamtrask and many others instructors in the community
In Lesson 5: Machine Learning Startups Need Structured Transparency and Consumers Need Structured Transparency Institutions 
Discussion was on the fact that "Data Owners maintain control over the only copies", Individuals and Organisation can get insights and access data without actually seeing the whole Data.
My question now is that how will Individuals/ Organisations be sure of the authenticity of the raw Data, where Collected Data or Aggregated Data Set are real/authentic not just fake data set. 😅
Data collectors can maintain fake dataset or include a large amount of unreal data just because they now they have control of the only copy for personal gain. This will make insights generated by individuals/organisation to be wrong

**Comment by hwrdtm** [comment, marked as answer]
Good question! You can imagine if one wishes to verify the authenticity of raw data, one solution would be to go and collect all of that data themselves. But this isn't practical!

Almost any other solution than that would require entities **trusting** one another - which is how much of society functions today - just like how we **trust** that scientists are speaking truth that certain vaccines are harmless, or how we **trust** the government is reporting accurately on unemployment, crime rates etc. Much of society functions like this today, ie. services provide simple-to-use abstractions because the value gain is in the form of greater societal productivity - Amazon is providing their AWS solutions so that developers all around the world can be more effective at leveraging cloud compute and storage capabilities, for example.

Given this, the focus is shifted from authenticating the data itself, to authenticating the _providers_ of the data themselves. When browsing the internet, this is when we need to make sure we are really talking to Amazon servers and not a man-in-the-middle, for example, and public key infrastructure and web certificates are a great solution to facilitate this.

That said, it is still important for these data owners / collectors to be audited to make sure they aren't producing fake data, and even this auditing process can be done in a privacy-preserving manner, as is described in the course in the output verification sections.

Hope this helps.
